pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -93,7 +92,7 @@ func (m *LoggerMiddleware) QueryData(ctx context.Context, req *backend.QueryData
 					"refID", refID,
 					"status", int(dr.Status),
 					"error", dr.Error,
-					"statusSource", pluginrequestmeta.StatusSourceFromPluginErrorSource(dr.ErrorSource),
+					"statusSource", statusSourceFromPluginErrorSource(dr.ErrorSource),
```

Comment:
```suggestion
					"statusSource", statusSourceFromPluginErrorSource(dr.ErrorSource),
					"target", m.pluginTarget(ctx, req.PluginContext),
```

Reply:
This is not relevant to your PR, but we could also add a target here, as we have that information and it can be helpful when searching for remote errors.

Reply:
Fixed